server/mscalendar/availability.go

```diff
@@ -157,7 +176,12 @@ func (m *mscalendar) setUserStatuses(users []*store.User, calendarViews []*remot
 			continue
 		}
 		if view.Error != nil {
-			m.Logger.Warnf("Error getting availability for %s. err=%s", user.Remote.Mail, view.Error.Message)
+			if numberOfLogs < logTruncateLimit {
+				m.Logger.Warnf("Error getting availability for %s. err=%s", user.Remote.Mail, view.Error.Message)
```

Comment:
Unrelated to the main purpose of this PR, but let's change this to the Mattermost user ID instead of the user's email. We shouldn't be logging this personal information, especially since it's coming from an external system.

Same with other occurrence of logging this value